inference-engine/tests/functional/plugin/conformance/subgraphs_dumper/include/ops_cache.hpp

```diff
@@ -0,0 +1,50 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#pragma once
+
+#include <utility>
+#include <vector>
+#include <string>
+#include <memory>
+#include <ngraph/ngraph.hpp>
+#include "matchers/matchers_manager.hpp"
+
+namespace SubgraphsDumper {
+
+class OPCache {
+public:
+    OPCache() : num_neighbours_to_cache(0) {
+        m_ops_cache = std::vector<std::pair<std::shared_ptr<ngraph::Node>, OPInfo>>();
+        manager = MatchersManager();
```

Comment:
Why initialization in c-tor body not in initialization list?

Reply:
Fixed